optionals/auto_load_settings_file/XEH_preInit.sqf

```diff
@@ -0,0 +1,12 @@
+#include "script_component.hpp"
+
+ADDON = false;
+
+// frame after preInit, but before postInit
+0 spawn {
```

Comment:
Maybe it would be safer and cleaner to have cba launch an event when preInit finishes